src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -52,7 +54,18 @@ public CommandResult execute() {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
             if (!Collections.disjoint(wordsInName, keywords)) {
                 matchedPersons.add(person);
+            } else {
+                Phone personsPhone = person.getPhone();
+                for (String substring: keywords) {
+                    if (substring.matches(Phone.PHONE_VALIDATION_REGEX)
+                            && !personsPhone.isPrivate()
+                            && personsPhone.containsSubstring(substring)) {
```

Comment:
Do you think the method name `getPersonsWithNameContainingAnyKeyword` is still correct? Consider splitting the phone check into another method.